plugins/apollo-explorer/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@backstage/plugin-apollo-explorer",
+  "version": "0.1.0",
```

Comment:
For new packages we start out at version `0.0.0` and then add a changeset with a `minor` bump. The changeset itself can give a brief introduction to the new package.